### ISSUE TITLE
No Picture

### ISSUE THREAD

**Issue by ITRecords38** [sammosna/profile.widget#1]
Hi,

i have must add my photo in a folder ?

Thanks

<img width="236" alt="profile" src="https://user-images.githubusercontent.com/19632490/88134998-4e9c3c80-cbe6-11ea-8666-df324dc8ed21.png">

**Comment by sammosna** [comment]
Hi, you have to run the `get_profile.sh` script

**Comment by ITRecords38** [comment]
> Hi, you have to run the `get_profile.sh` script

Hi,
yes
<img width="1026" alt="ITRecords 1" src="https://user-images.githubusercontent.com/19632490/88692564-7e73a480-d0fe-11ea-8fb6-5fc5cdb10e49.png">

**Comment by sammosna** [comment]
Try running this:
```bash
DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
dscl . -read $HOME JPEGPhoto | tail -1 | xxd -r -p > "$DIR/profile.jpg"
```

**Comment by ITRecords38** [comment]
> Try running this:
> 
> ```shell
> DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
> dscl . -read $HOME JPEGPhoto | tail -1 | xxd -r -p > "$DIR/profile.jpg"
> ```

Same problem :(

**Comment by coolajami** [comment]
same problem here, any chance to have to do with the MacOS version? I'm running Catalina.

**Comment by sammosna** [comment]
@coolajami I'm on Big Sur beta... But I don't think it depends on that.
[Here](https://www.hawkdive.com/all-admin-accounts-changed-to-standard/) they say that `This error states the admin group is missing, which would explain the problem. If the admin group is not available, then the system cannot add new users to it and therefore will not promote any to have administrative functions.` So probably it's a permission issue. 
@coolajami @GrimlocK38  Try this (changing `<username>` with your username): 
```bash
dscl . -read /Users/<username> JPEGPhoto | tail -1 | xxd -r -p > "/Users/<username>/Desktop/profile.jpg"
```
Does it create a `profile.jpg` on your Desktop?

**Comment by ITRecords38** [comment]
Hi,

after testing this modification it did not generate the photo on the desktop. Grrrrr 🤪

`Does it create a profile.jpg on your Desktop?`

**Comment by coolajami** [comment]
> @coolajami I'm on Big Sur beta... But I don't think it depends on that.
> [Here](https://www.hawkdive.com/all-admin-accounts-changed-to-standard/) they say that `This error states the admin group is missing, which would explain the problem. If the admin group is not available, then the system cannot add new users to it and therefore will not promote any to have administrative functions.` So probably it's a permission issue.
> @coolajami @GrimlocK38 Try this (changing `<username>` with your username):
> 
> ```shell
> dscl . -read /Users/<username> JPEGPhoto | tail -1 | xxd -r -p > "/Users/<username>/Desktop/profile.jpg"
> ```
> 
> Does it create a `profile.jpg` on your Desktop?

No, sorry, tried several combinations of the folder path, still nothing :(.

**Comment by lillycham** [comment]
I'm having this issue on Big Sur (Intel, 11.6), except I do have the profile.jpg... 
I found that changing ```src="/profile.widget/profile.jpg"``` to ```src="profile/profile.jpg"``` in index.jsx fixed it on my end.

**Comment by umar8092** [comment]
Guys i solved it. 
Just open terminal, 

change the location using cd to whereever you want (Like the desktop)
and run this script 
```
#!/bin/bash
DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
dscl . -read $HOME JPEGPhoto | tail -1 | xxd -r -p > "$DIR/profile.jpg"
```

Your picture will be saved in that location.

Copy this picture to widget folder of Übersicht.

Paste. Open the index in any notepad and change the src to 

`src="profile.jpg"`
